tests/torch/ptq/test_wrap_model.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2023 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import torch
+from torch import nn
+
+from nncf.torch.model_creation import wrap_model
+
+
+class TestArgumentModel(nn.Module):
```

Comment:
The classes that do not group test cases should not start with `Test`, otherwise pytest will try to execute the `test_*` methods if there are any present in such a class and will also complain that such a class should not contain an `__init__` method.

Reply:
Thanks for the comment. I removed `Test` from class names.